imagebuildah/build.go

```diff
@@ -1199,7 +1199,7 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options BuildOpt
 				contents.Close()
 				return errors.Wrapf(err, "error reading info about %q", dfile)
 			}
-			if dinfo.Size() == 0 {
+			if dinfo.Mode().IsRegular() && dinfo.Size() == 0 {
```

Comment:
Is this needed for this change, or just an unrelated tweak?

Reply:
Needed, this is why your tests with -f /dev/stdin were not working.
Size of a pipe is 0, If we only check on size of files, then the pipe will go into the imagebuilder and the build succeeds.

Reply:
thx for the 411